shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -27,11 +27,11 @@
     case UIAccessibilityScrollDirectionRight:
```

Comment:
Can you also update the comment above? It appears that only up/down are switched?